### PR TITLE
docs: update auth options

### PR DESCRIPTION
**Pull request by jannyHou**
Fixes: https://github.com/strongloop/loopback-next/issues/4576

Copy from ^

I verified with an app in PR https://github.com/strongloop/loopback-next/pull/4571

```ts
const auth_options: AuthorizationOptions = {
      defaultDecision: AuthorizationDecision.ALLOW,
      precedence: AuthorizationDecision.ALLOW,
    }
    
this.configure(AuthorizationBindings.COMPONENT).to(auth_options);
```
works.
The debug log:
```
loopback:authorization:interceptor Authorization options { defaultDecision: 'Allow', precedence: 'Allow' } +272ms
```
And according to the code in
https://github.com/strongloop/loopback-next/blob/6eea5e428b145cafb84a998bd53979da8c8fba07/packages/authorization/src/authorize-interceptor.ts#L38

Your solution is the correct fix. Submitting a PR to update the doc

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Loopback-component-authorization.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ class.
     defaultDecision: AuthorizationDecision.DENY,
   };
 
-  const binding = app.component(AuthorizationComponent);
-  app.configure(binding.key).to(options);
+  app.component(AuthorizationComponent);
+  app.configure(AuthorizationBindings.COMPONENT).to(options);
```
Comment (derdeka) [inline docs/site/Loopback-component-authorization.md]: Are you sure that `configure` after `component` works? On other places we have to do `configure` before `component` to make it working?

Comment (jannyHou) [reply docs/site/Loopback-component-authorization.md]: @derdeka I wasn't sure, so I verified with app in https://github.com/strongloop/loopback-next/pull/4571, you can read the PR description to see my testing result.

`this.configure()` is called in the app constructor, and whenever the authorization interceptor is invoked, the options will be injected. So I believe after app runs, making API calls will inject the configured options.

Comment (derdeka) [reply docs/site/Loopback-component-authorization.md]: I would like to suggest to switch the order to have it consistent with other docs.

Comment (jannyHou) [reply docs/site/Loopback-component-authorization.md]: @derdeka 👌 switched the order to have the configure before mounting component.

**Pull request merged**